mayland-blocks/child-theme.json

```
@@ -79,7 +79,8 @@
 				"selection": "var(--wp--preset--color--selection)"
 			},
 			"margin": {
-				"horizontal": "32px"
+				"horizontal": "32px",
```

Should mayland blocks get the responsive value also? e.g. 

```suggestion
				"horizontal": "min(32px, 5vw",
```

---

Sure.